### ISSUE TITLE
[Suggestion] Auto Pearl

### ISSUE THREAD

**Issue by kandda31** [MeteorDevelopment/meteor-client#2470]
### Describe the feature

**Auto Pearl**

_Automatically throws a pearl from your inventory on a surface before falling into the void. Useful for knockback servers._

**Options Suggestions:**

Only When Look: ■

Only When Click: ■

Rotate: ■

Rotate Speed: <value> [Determines how fast you should rotate towards the target]

Distance: <value> [The amount of blocks from the edge to the opposite way of it]

Target: [Block]  [Determines the type of target]
            [Player] 

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client.
(I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested.
(I have checked every suggestion on github)

- [X] This feature is a valid suggestion.
(I have read the FAQs and Guides)

**Comment by Cypphi** [comment]
This client is not for cheating on servers that don't allow it

**Comment by kandda31** [comment]
> This client is not for cheating on servers that don't allow it

First of all, not every server has an anti cheat system. And i never saw a module like that anywhere so it will surely bypass the anticheat if there is one.

**Comment by Cypphi** [comment]
> > This client is not for cheating on servers that don't allow it
> 
> First of all, not every server has an anti cheat system. And i never saw a module like that anywhere so it will surely bypass the anticheat if there is one.

I don't think you get the idea of a utility mod. This is a **utility mod** aimed at anarchy servers. This client is not made to be used on any server which does **not** allow cheating. No matter if they do or do not have an anticheat. And to be fair, you should be using something else than meteor for pvp.........

**Comment by kandda31** [comment]
> > > This client is not for cheating on servers that don't allow it
> > 
> > 
> > First of all, not every server has an anti cheat system. And i never saw a module like that anywhere so it will surely bypass the anticheat if there is one.
> 
> I don't think you get the idea of a utility mod. This is a **utility mod** aimed at anarchy servers. This client is not made to be used on any server which does **not** allow cheating. No matter if they do or do not have an anticheat. And to be fair, you should be using something else than meteor for pvp.........

So you will tell me that there isn't any hack in the client lmao
